src/AppBundle/Service/EventProcessor.php

```diff
@@ -348,18 +348,19 @@ private function setContributionsWikipedias(EventWiki $wiki, EventWikiRepository
         $wiki->setPagesEdited($pageIdsEdited);
 
         $pageIds = array_merge($pageIdsCreated, $pageIdsEdited);
-        $ret = $this->eventRepo->getEditStats($dbName, $pageIds, $start, $end, $usernames);
-
         $diff = $ewRepo->getBytesChanged($this->event, $dbName, $pageIds, $usernames);
 
-        $this->pagesCreated += $ret['created'];
-        $this->pagesImproved += $ret['edited'];
-        $this->edits += $ret['edits'];
+        $totalCreated = count($pageIdsCreated);
+        $totalEdited = count($pageIdsEdited);
+        $total = count($pageIds);
```

Comment:
What about `$totalCreated + $totalEdited`, then we can remove line 350?

Reply:
I think we still need $pageIds to get the bytes changed. And I've just readded getEditStats too...